docs/fastdds/transport/tcp/tls.rst

```diff
@@ -127,6 +128,12 @@ The following table describes the data members that are configurable on |TCPTran
      - |TCPTransportDescriptor::TLSConfig::TLSHandShakeRole::DEFAULT-api|
      - Role that the transport will take on handshaking.
        See |TLSRole|.
+   * - |TCPTransportDescriptor::TLSConfig::server_name-api|
+     - ``string``
+     - ``""``
+     - (UNIMPLEMENTED) Server name or host name  |br|
```

Comment:
We normally prefer `[*]` (see [here](https://fast-dds.docs.eprosima.com/en/latest/fastddsgen/dataTypes/dataTypes.html#annotations))
```suggestion
     - [UNIMPLEMENTED] Server name or host name  |br|
```